apps/webapp/src/components/shared/input-token.tsx

```diff
@@ -67,24 +93,17 @@ export default function InputToken({
       </div>
 
       <div className='mt-[6px] flex items-center justify-between gap-2'>
-        <p
-          className={cn(
-            'break-all md:test-[12px] xl:text-base font-bold text-light-brown',
-            value && 'text-muted-foreground',
+        <div className='flex items-start gap-2'>
+          {totalGasPriceAsString && (
+            <>
+              <img src='/fuel.svg' alt='Gas price' className='h-5 w-5' />
+              <p className='font-bold text-muted-foreground'>{totalGasPriceAsString}</p>
+            </>
           )}
-        >
-          ${displayAmount(Number(value) * tempPrice)}
```

Comment:
As far as I can tell, this appeared to be a stub of the gas price. (I'm not sure of that, though — I'm not actually sure what `tempPrice` was for.) So I removed it and displayed the gas price in its place. If I'm missing something about the `tempPrice`, let me know.